lib/cryptor.rb

```diff
@@ -8,19 +8,18 @@ def initialize(message)
 
   def message_in_alphabet_positions
     alphabet = ('a'..'z').to_a << ' '
-    @message.split('').map {|letter| alphabet.find_index(letter)}
+    message.split('').map {|letter| alphabet.find_index(letter)}
```

Comment:
removing the `@` allows to stub out the reader method for the corresponding instance variable